dotnet/notebooks/05-using-function-calling.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to the Planning/Function Calling\n",
+    "\n",
+    "Once you have multiple plugins, you then need a way for your AI agent to use them together to solve a user’s need. This is where planning comes in.\n",
+    "\n",
+    "Early on, Semantic Kernel introduced the concept of planners that used prompts to request the AI to choose which functions to invoke. Since Semantic Kernel was introduced, however, OpenAI introduced a native way for the model to invoke or “call” a function: function calling. Other AI models like Gemini, Claude, and Mistral have since adopted function calling as a core capability, making it a cross-model supported feature.\n",
+    "\n",
+    "Because of these advancements, Semantic Kernel has evolved to use function calling as the primary way to plan and execute tasks.\n",
```

Comment:
I don't think we need to explain the history as that information isn't relevant for someone who is new. 

The Blog post is there for people who are migrating.